src/init.d/image_checker.py

```diff
@@ -86,7 +86,8 @@ def __init__(self, job_config, secret):
         image_info = docker_images[0]
 
         self._image_uri = image_info["uri"]
-        self._registry_uri = DEFAULT_REGISTRY
+        self._registry_uri = self._get_registry_from_image_uri(
```

Comment:
why use https as default schema? there's no way to include schema in image uri

Reply:
By default, will use `https`. If use registry use `http`, this check will be ignored.